packages/fsml/fsml.0.2a/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+synopsis:
+  "A library for describing and describing synchronous finite state machines"
+description:
+  "A library for describing and describing synchronous finite state machines"
```

Comment:
This field is duplicated with `synopsis`. It's not a big deal but just in case you happen to also modify the file, this one can be removed

Reply:
Done